pkg/users/account-create-dialog.js

```diff
@@ -222,57 +301,60 @@
 
         errs.user_name = validate_username(user_name, accounts);
 
-        return password_quality(password, force)
-                .catch(ex => {
-                    errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
-                })
+        const promises = [
+            password_quality(password, force)
+                    .catch(ex => {
+                        errs.password = (ex.message || ex.toString()).replaceAll("\n", " ");
+                    }),
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230131-135835-5b43f8e5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)